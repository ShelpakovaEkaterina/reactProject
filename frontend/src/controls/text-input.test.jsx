import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from './text-input';

test('text-input', () => {
    const component = renderer.create(
        <TextInput input={{ name: 'test' }} type="input" label="test" meta={{}} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

