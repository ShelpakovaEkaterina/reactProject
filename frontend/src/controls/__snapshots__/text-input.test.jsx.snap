// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text-input 1`] = `
<div
  className="text-input text-input_input"
>
  <label
    className="text-input__label"
    htmlFor="test"
  >
    test
  </label>
  <input
    className="text-input__input"
    id="test"
    name="test"
    type="input"
  />
</div>
`;
